.github/workflows/trigger-e2e-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Trigger E2E tests Workflow
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        required: true
+      dispatch_id:
+        description: 'Dispatch Id (repo/sha)'
+        required: true
+
+jobs:
+  trigger_e2e_tests:
+    name: "Trigger E2E tests Workflow"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Trigger E2E tests"
+        uses: actions/github-script@v5
+        env:
+          version: ${{ inputs.version }}
+          dispatch_id: ${{ inputs.dispatch_id }}
+        with:
+          github-token: ${{ secrets.GH_REPOSITORY_ADMIN_TOKEN }}
+          script: |
+            const {repo, sha, version} = process.env;
+            const owner = context.payload.repository.organization
+            const e2eRepo = 'e2e-system-tests'
+            const workflow_id = 'frontegg-react-e2e-tests.yml'
+            const context = `${owner}/${e2eRepo}`
+            const dispatch_id = `${repo}/${sha}`
+
+            const res = await github.rest.repos.createCommitStatus({
```

Comment:
There are actions to do it

Reply:
After we finish building our Frontegg CI App, this code will be replaced with more elegant way